tasks/section_10_level1.yml

```diff
@@ -98,22 +98,18 @@
       - section10.4
 
   - name: 10.5.1 Lock Inactive User Accounts (check) (Scored)
-    command: grep INACTIVE /etc/login.defs
+    command: bash -c "useradd -D | grep -oP '(?<=^INACTIVE=)-?[0-9]+'"
```

Comment:
Yes, the `command` module will execute the process, and using the `shell` you will wrap it around with `bash` as you did. So yes, using the `shell` module is better here to keep the code simple.